### ISSUE TITLE
Updating the AWS PIA to incorporate new region flexibility

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#1772]
**Describe the issue**
We will need to update our AWS Program PIA for the ECF Platform to include the use cases in regions outside of Canada.

**Additional context**
- https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/privacy/privacy-impact-assessments/guidance-for-privacy-impact-assessments#part-four
![PIA process clear.drawio.png](https://images.zenhubusercontent.com/5e98c26e92d4be32d8e29cb0/752fa133-6f0b-4792-8249-e88c76017b71)
- The linked tickets are related but nothing is actually blocking this ticket (ZenHub only has the one linkage type)

This is lower priority so if this drops out of sprint that's ok.

**Definition of done**
- Find the old existing current PIA for AWS 
- Gain understanding of the PIA non-Canadian PIA + Assessment process (see link above)
- Gain understanding of the technical work in the two linked tickets
- Draft new slightly updated version of the AWS Program PIA as a document stored in the CPF Teams files (pull down a copy, store it as a Word doc in Teams under CPF FTE Internal / 004 - Compliance / Privacy / Draft PIA Updates)
- Review proposed changes with PO

**Comment by bruce-wh-li** [comment]
Existing PIA for AWS can be found under Compliance > Privacy

**Comment by garymatics** [comment]
PIA draft shared for review/feedback.

**Comment by garymatics** [comment]
Met with Julian regarding value/priority of this ticket. Julian has confirmed that this remains a valuable ticket and accomplishing this ticket/work will partially unblock the ability to meet the new FOIPPA configurations.  Julian has said that this ticket originated from Gen and he confirmed that this ticket continues to be of value.

**Comment by garymatics** [comment]
This ticket will be addressed further in detail once there is further clarity about business needs to store BC Gov data in outside Canada regions.
